app/templates/src/app/main/__angular-material.html

```diff
@@ -1,8 +1,9 @@
 <div layout="vertical" layout-fill>
-
-  <header ng-include="'components/navbar/navbar.html'"></header>
-
   <md-content>
+    <md-toolbar>
+      <header ng-include="'components/navbar/navbar.html'"></header>
```

Comment:
Could be, but I'm not sure what you are asking.  Moving the original header into a md-toolbar tag made the spacing for the page content drop below the header instead of writing over it.